### ISSUE TITLE
Strange unfocus after file operations

### ISSUE THREAD

**Issue by dstd** [trol73/mucommander#166]
After a file operation (like a copying, deleting) I get a strange focus loss.
I'm not sure when it happened, but suspect it was the update to MacOS High Sierra.

After a file ops, current file pointer in a file list becomes gray (pictured below), like when a dialog is above the file. And in that state there's no response on Up/Down keys, yet I still can move the pointer by mouse clicks, but only in current panel, the opposite panel is not interactive. App's windows list contains only main window of the app, and all menu items are disabled.

This state can be dismissed by switching into another app and return back into Commander.

Any ideas how this could be fixed?

![5f5941e8-a996-4f9f-baa1-dd920dca5d23](https://user-images.githubusercontent.com/1444844/46439395-28837f00-c769-11e8-98d1-acf0ef7fa893.png)

**Comment by trol73** [comment]
Hm.. I can't reproduce that now on Mojave (and never seen before on High Sierra)

**Comment by bagonyi** [comment]
I have very similar issues on:

```
Java
Runtime version: 1.8.0_181
VM name: Java HotSpot(TM) 64-Bit Server VM
VM version: 25.181-b13
VM vendor: Oracle Corporation

OS
Name: Mac OS X
Version: 10.13.3
Architecture: x86_64
```

I can replicate this issue every time by doing the following:

1. Start trolCommander
2. Switch to an other application and back to trolCommander (Cmd+Tab twice)
3. Press F7 to create a directory, enter a name and press enter
4. Focus is lost, I have to Cmd+Tab twice to get the focus back

Interestingly, if I start trolCommander and I don't switch to an other application, I can create as many directories I want, the focus won't be lost. It looks like you need to make trolCommander's main window to lose focus to trigger this issue.

I am using https://github.com/keycastr/keycastr to show what keys I am pressing on my keyboard:

![trolcommander-focus-issue](https://user-images.githubusercontent.com/604444/47464960-de7b3f80-d7e2-11e8-9976-3d673bd85795.gif)

**Comment by trol73** [comment]
I can't reproduce that either on Mojave and last nightly build (from Google Drive). But I found other focus issue - focus disappears after pressing F8 on the ".." (parent) directory

**Comment by bagonyi** [comment]
I have managed to make this issue disappear. After removing all JREs / JDKs from my computer in `/Library/Java/JavaVirtualMachines/` I realised there is still some traces of Java left on my system in `/Library/Internet Plug-Ins/JavaAppletPlugin.plugin` (Java 10). After deleting this and `/Library/PreferencePanes/JavaControlPanel.prefPane`, then reinstalling Oracle JDK 8, now everything works as normal.
